admin-app/image.go

```diff
@@ -21,6 +25,53 @@ var allowed_content_types = map[string]bool{
 	"image/jpeg": true, "image/png": true, "image/gif": true,
 }
 
+func resizeImage(srcPath string, width uint) error {
```

Comment:
I'd like to see this function taking `srcPath` and `dstPath`. The logic here is:

+ We leave the original image saved, because we still want to show them in high res when you click on them in the gallery.
+ We want to save the new, smaller images so we can use in thumbnails!

So basically, write the resized image to a new file. Like `<UUID>_small.jpg` in the same directory.